examples/now/pages/index.vue

```diff
@@ -135,6 +140,9 @@ export default {
     isCheckbox(type) {
       return type === 'checkbox'
     },
+    deleteFile() {
+      this.file = null
+    },
```

Comment:
IMO it's unecessary to add another method, you can just add
```vue
<button @click="file = null" />

Reply:
Done.